src/Sulu/Bundle/ContentBundle/Resources/views/Template/content-types/url.html.twig

```diff
@@ -1,13 +1,16 @@
 <div id="{{ id|raw }}"
      data-property='{{ property|json_encode }}'
      data-mapper-property="{{ property.name }}"
-     data-aura-component="input@husky"
-     data-aura-skin="url"
+     data-aura-component="url-input@husky"
      data-aura-instance-name="{{ id|raw }}"
-     data-aura-input-id="{{ id|raw }}"
-     data-type="husky-input"
+     data-aura-input-class="no-preview-update"
+     data-aura-scheme="{{ params.defaults.value.scheme.value | default('http://') }}"
+     data-aura-specific-part="{{ params.defaults.value.specific_part.value | default('') }}"
+     data-aura-schemes="{{ sulu_parameter_to_names_array(params.schemes.value) | json_encode }}"
+     data-type="url-input"
+     data-type-schemes="{{ sulu_parameter_to_names_array(params.schemes.value) | json_encode }}"
      data-form="true"
-     class="preview-update trigger-save-button"
+     class="preview-change-update trigger-save-button"
```

Comment:
Ok, now I get it... maybe we should an issue targeted at 2.0, which will replace the `preview-update` with the `preview-change-update` class, so that we handle all of this stuff the same.